applications/rag/README.md

```diff
@@ -29,22 +29,6 @@ Install the following on your computer:
 * [Helm](https://helm.sh/docs/intro/install/)
 * [Gcloud](https://cloud.google.com/sdk/docs/install)
 
-### Bring your own cluster (optional)
```

Comment:
I think it's fine to keep this because it's marked optional. The terraform route should be flexible enough to support this option for really advanced users IMO. But I think it can be even clearer that most users should use new clusters instead

Reply:
Yeah, I want to keep this. I'm ok to simplify to a gcloud step once we switch back to Autopilot.